src/GetLanguages.js

```diff
@@ -66,14 +68,14 @@ class GetLanguages extends Component {
     }
 
     $.ajax({
-      url: "https://api.mt2414.in/v1/get_books",
+      url: "http://127.0.0.1:8000/v1/get_books",
       contentType: "application/json; charset=utf-8",
       data: JSON.stringify(data),
       method : "POST",
       headers: {
                 "Authorization": "bearer " + JSON.stringify(accessToken['access_token']).slice(1,-1)},
       success: function (result) {
-        var getBook = JSON.parse(result);
+        // var getBook = JSON.parse(result);
```

Comment:
Because till now getBook api is not completed at our end. Right now I have removed the commented part.